### PR TITLE
Added pxhd expiry

### PR DESCRIPTION
**Pull request by alexbpx**

### CODE REVIEW

File: perimeterx/px_utils.py [hunk 1]
```diff
@@ -51,5 +51,20 @@ def prepare_custom_params(config, dict_to_add):
...
+
+weekdayname = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
+
+monthname = [None,
+             'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+             'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+
+
+def getExpiryDate(future=31536000):
+    from time import gmtime, time
```
Comment (ilaipx) [inline perimeterx/px_utils.py]: why import inside the function?

File: perimeterx/middleware.py [hunk 2]
```diff
@@ -91,11 +91,9 @@ def config(self):
 
 def create_custom_pxhd_callback(context, start_response):
     def custom_start_response(status, headers, exc_info=None):
-        if context.pxhd:
-            pxhd_cookie = SimpleCookie()
-            pxhd_cookie['_pxhd'] = ""
-            pxhd_cookie['_pxhd']['expires'] = 365 * 24 * 60 * 60
-            headers.append(('Set-Cookie', "_pxhd={}; path=/; expires={}".format(context.pxhd, pxhd_cookie['_pxhd']['expires'])))
+        if not context.pxhd or (context.response_pxhd and context.pxhd != context.response_pxhd):
```
Comment (ilaipx) [inline perimeterx/middleware.py]: Maybe we can use the Cookie module (import Cookie) like:
```python
from Cookie import SimpleCookie
c = SimpleCookie()
c['_pxhd'] = 'xxx'
c['_pxhd']['path'] = '/'
c['_pxhd']['expires'] = 12 * 30 * 24 * 60 * 60 #  1 year
```
Looks cleaner imo.

Comment (alexbpx) [reply perimeterx/middleware.py]: No.

Comment (alexbpx) [reply perimeterx/middleware.py]: The string representation of SimpleCookie isn't suitable for us

**Pull request merged**